FirebaseAuth/Sources/AuthProvider/Phone/FIRPhoneAuthProvider.m

```diff
@@ -593,6 +595,16 @@ - (void)verifyClientAndSendVerificationCodeToPhoneNumber:(NSString *)phoneNumber
  */
 - (void)verifyClientWithUIDelegate:(nullable id<FIRAuthUIDelegate>)UIDelegate
                         completion:(FIRVerifyClientCallback)completion {
+  // Remove the simulator check below after FCM supports APNs in simulators
+  if (@available(iOS 16, *)) {
+    if ([GULAppEnvironmentUtil isSimulator]) {
```

Comment:
You can use `#if TARGET_OS_SIMULATOR` so this bit of code doesn't get compiled into non-simulator target destination builds.

Reply:
Yeah, good point